internal/saucecloud/cloud.go

```diff
@@ -170,6 +170,11 @@ func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, interrupted bool
 	// FIXME framework specifics shouldn't be handled in a generic package. Could simply do an empty string check instead.
 	if opts.Framework == "espresso" {
 		l.Str("deviceName", opts.DeviceName).Str("platformVersion", opts.PlatformVersion).Str("deviceId", opts.DeviceID)
+		if isRDC {
+			if opts.DevicePrivateOnly {
+				l.Str("private", "true")
+			}
+		}
```

Comment:
I'd just always log it in that case:
```suggestion
		if isRDC {
			l.Bool("private", opts.DevicePrivateOnly)
		}
```

Further note (to us): The upper block should be refactored, because right now, this info is only printed out for espresso. Pretty sure this information is interesting for `xcuitest` as well (which is pretty much why framework specific code in this location is bad, see my `FIXME` in line 170 💡 We can take care of that, or, should you feel adventurous, you can make the change as well :)

Reply:
I'll fix it for my use case ;)

Thanks for the further suggestion!